### PR TITLE
Export operations

### PR DESCRIPTION
**Pull request by marten**
Extract the handling of various project exports to operation classes. More lines of code, but faster to test because we don't need the whole controller layer.

# Review checklist

* First, the most important one: is this PR small enough that you can actually review it? Feel free to just reject a branch if the changes are hard to review due to the length of the diff.
* If there are any migrations, will they the previous version of the app work correctly after they've been run (e.g. the don't remove columns still known about by ActiveRecord).
* If anything changed with regards to the public API, are those changes also documented in the `apiary.apib` file?
* Are all the changes covered by tests? Think about any possible edge cases that might be left untested.

### CODE REVIEW

File: spec/operations/projects/create_classifications_export_spec.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+require 'spec_helper'
+
+describe Projects::CreateClassificationsExport do
+  let(:user) { create :user }
+  let(:api_user) { ApiUser.new(user) }
+  let(:operation) { described_class.with(api_user: api_user) }
+  let(:project) { create(:full_project, owner: user) }
+
```
Comment (camallen) [inline spec/operations/projects/create_classifications_export_spec.rb]: agree with hound on this

Comment (marten) [inline spec/operations/projects/create_classifications_export_spec.rb]: Meh. I had to disable my editor config when I made a change to talk. Will fix.

**Comment by camallen** [comment]
LGTM - slowly shrinking the controllers and getting the services pattern rolled out. Can we hold of merging this till next week?

**Comment by marten** [comment]
Sure.

**Pull request merged**